shardingsphere-infra/shardingsphere-infra-federation/shardingsphere-infra-federation-executor/src/main/java/org/apache/shardingsphere/infra/federation/executor/customized/AbstractUnsupportedUpdateOperationResultSet.java

```diff
@@ -0,0 +1,451 @@
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateInt(final int columnIndex, final int x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateInt(final String columnLabel, final int x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateLong(final int columnIndex, final long x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateLong(final String columnLabel, final long x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateFloat(final int columnIndex, final float x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateFloat(final String columnLabel, final float x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateDouble(final int columnIndex, final double x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateDouble(final String columnLabel, final double x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateBigDecimal(final int columnIndex, final BigDecimal x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateBigDecimal(final String columnLabel, final BigDecimal x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateString(final int columnIndex, final String x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateString(final String columnLabel, final String x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateNString(final int columnIndex, final String nString) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateNString(final String columnLabel, final String nString) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
+    
+    @Override
+    public final void updateBytes(final int columnIndex, final byte[] x) throws SQLException {
+        throw new SQLFeatureNotSupportedException("updateXXX");
+    }
```

Comment:
Is it better to change `updateXXX` to a specific method name?

Reply:
I'm afraid we can't. Because update***() are defined in java.sql.ResultSet.

Reply:
> I'm afraid we can't. Because update***() are defined in java.sql.ResultSet.

I mean `updateXXX` in `throw new SQLFeatureNotSupportedException("updateXXX");`

Reply:
so ga, I'll fix this.